tests/test_validators.py

```diff
@@ -22,6 +22,9 @@
 )
 from pydantic.decorators import root_validator
 
+# TODO: do we really want this API? Can we change it?
+RootValidatorValuesAndFieldsSet = Tuple[Dict[str, Any], Set[str]]
```

Comment:
I would suggest we keep it for now and get this merged, but revisit before the alpha release.